ppr-ui/src/composables/parties/useSecuredParty.ts

```diff
@@ -20,7 +21,7 @@ const initAddress = {
 }
 const { isPartiesValid } = useParty()
 
-export const useSecuredParty = (props, context) => {
+export const useSecuredParty = (props?, context?) => {
```

Comment:
If `props` and `context` are optional, we probably should set optional chaining for them on line 63, 92?

Reply:
Ill push the commit soon, but I removed props, and just passed activeIndex to the functions that used it, then optionally chained context.